### PR TITLE
Add EEG talk to tutorials

### PR DESCRIPTION
**Pull request by phackstock**
Added my talk last week at EEG to the tutorial section.

Hope this works.

### CODE REVIEW

File: tutorials/eeg-coding-culture.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Coding culture at EEG
+=====================
+
+This talk was given by `Philip Hackstock <https://iiasa.ac.at/staff/philip-hackstock>`_
+on the 17th of January 2022 for the EEG group at the TU Vienna.
```
Comment (danielhuppmann) [inline tutorials/eeg-coding-culture.rst]: You should introduce the abbreviation. Also, adding a link to the EEG website would be helpful for context.
```suggestion
on the 17th of January 2022 for the *Energy Economics Group* (EEG) group at the TU Vienna.
```

File: tutorials/eeg-coding-culture.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Presentation slides (.pptx & .pdf, english): DOI: `10.5281/zenodo.5881657
+<https://doi.org/10.5281/zenodo.5881657>`_
```
Comment (LauWien) [inline tutorials/eeg-coding-culture.rst]:
```suggestion
- DOI: `10.5281/zenodo.5881657 <https://doi.org/10.5281/zenodo.5881657>`_
```
But I guess this suggestion is only personal preference to stay in the same layout like "Example repositories:"

File: tutorials/eeg-coding-culture.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Coding culture at EEG
+=====================
+
+This talk was given by `Philip Hackstock <https://iiasa.ac.at/staff/philip-hackstock>`_
+on the 17th of January 2022 for the EEG group at the TU Vienna.
+
+Presentation slides (.pptx & .pdf, english): DOI: `10.5281/zenodo.5881657
```
Comment (LauWien) [inline tutorials/eeg-coding-culture.rst]:
```suggestion
Presentation slides (.pptx & .pdf, english):
```

**Review by LauWien — COMMENTED** [review]
Just a few suggestions in line.

**Review by LauWien — APPROVED** [review]
Perfect, thank you!

**Pull request merged**